questionnaire/2020/questions.yaml

```diff
@@ -403,19 +406,22 @@
     id: trend-tools
     responses: 
     - { text: Ansible, id: ansible }
+    - { text: Terraformm, id: terraform }
     - { text: Saltstack, id: saltstack }
     - { text: NAPALM, id: napalm }
     - { text: Nornir, id: nornir }
     - { text: Netbox, id: netbox }
     - { text: ELK (Logstash/Kibana), id: elk }
-    - { text: Grafana/Prometheus/Influxdb, id: tsdb }
+    - { text: Grafana, id: grafana }
+    - { text: Prometheus, id: prometheus }
+    - { text: Influxdb, id: influxdb }
     - { text: Kentik, id: kentik }
     - { text: NSO (tail-F), id: nso }
     - { text: Git (github/gitlab), id: git }
-    - { text: "Network Verification Software (Batfish, Forward Networks, Veriflow ..)", id: network-verification }
-    - { text: "Keyword based testing (RobotFramework, Behave)", id: keyword-testing }
-    - { text: "CI (jenkins, travis-ci, circle-ci, gitlab-ci)", id: ci }
     - { text: Stackstorm, id: stackstorm }
+    - { text: GNS3, id: gns3 }
+    - { text: EVE-NG, id: eve-ng }
+    
```

Comment:
We're listing these tools under the virtual environment section, I don't think we need to single them out and list them here also

Reply:
Personally I like these 2 questions because they are giving another perspective on these tools. For example, last year it was very interesting to see that 40% of the participants are thinking about using Nornir. 
Most if not all of the other tools listed in here are also mentioned in other questions in the survey like Napalm, Nornir, Ansible etc ..

Since GNS3 and EVE-NG were leading the responses around virtual env last year, I thought it would be good to add them here to have a different perspective. 
It was the same idea behind Grafana, prometheus and influx. I'll love to have more details on how these solutions are trending right now